### PR TITLE
Harvester: Fix reserved memory edit issue

### PR DESCRIPTION
**Pull request by n313893254**
### Summary
Fixes reserved memory edit issue

### Occurred changes and/or fixed issues

- https://github.com/harvester/harvester/issues/2589

### Technical notes summary

~~When the VM page is updated, the unit input is also updated, which will show the last value instead of the value that is being delayed input.~~

### CODE REVIEW

File: shell/mixins/harvester-vm/index.js [hunk 1]
```diff
@@ -1250,7 +1250,13 @@ export default {
           }
         });
       }
-    }
+    },
+
+    updateReserved(value = {}) {
+      const { memory } = value;
+
+      this.$set(this, 'ReservedMemory', memory);
```
Comment (WuJun2016) [inline shell/mixins/harvester-vm/index.js:1258]: ReservedMemory => reservedMemory

File: shell/edit/kubevirt.io.virtualmachine/VirtualMachineReserved.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  <div class="row mb-20">
+    <div class="col span-6">
+      <UnitInput
+        v-model="memory"
+        v-int-number
+        :label="t('harvester.virtualMachine.input.reservedMemory')"
+        :mode="mode"
+        :input-exponent="2"
+        :increment="1024"
+        :output-modifier="true"
```
Comment (WuJun2016) [inline shell/edit/kubevirt.io.virtualmachine/VirtualMachineReserved.vue:49]: @input="change"

File: shell/mixins/harvester-vm/index.js (hunk 1 above)
Comment (n313893254) [reply shell/mixins/harvester-vm/index.js]: Done

File: shell/edit/kubevirt.io.virtualmachine/VirtualMachineReserved.vue (hunk 2 above)
Comment (n313893254) [reply shell/edit/kubevirt.io.virtualmachine/VirtualMachineReserved.vue:49]: Done

**Review by WuJun2016 — APPROVED** [review]
LGTM

**Pull request merged**